src/react-apps/applications/runtime/src/containers/Toolbar.tsx

```diff
@@ -138,37 +138,35 @@ class ToolbarClass extends React.Component<IToolbarProps, IToolbarState> {
     );
```

Comment:
Could probably just delete the whole file here, since it's not in use in runtime.

Reply:
Toolbar and dependencies deleted